src/modules/initcpiocfg/main.py

```diff
@@ -21,20 +21,19 @@
 #   You should have received a copy of the GNU General Public License
 #   along with Calamares. If not, see <http://www.gnu.org/licenses/>.
 
-import libcalamares
 from libcalamares.utils import debug, target_env_call
 import os
 from collections import OrderedDict
 
 import gettext
-_ = gettext.translation("calamares-python",
+translate = gettext.translation("calamares-python",
                         localedir=libcalamares.utils.gettext_path(),
```

Comment:
How is `libcalamares.utils` still in-scope as a name here?

Reply:
It's not:

```
14:39:57 [6]: virtual Calamares::JobResult Calamares::PythonJob::exec()
    Job file "/home/adridg/src/calamares/build/src/modules/initcpiocfg/main.py" 
Python Error:
 <class 'NameError'> 
 name 'libcalamares' is not defined 
 File "/home/adridg/src/calamares/build/src/modules/initcpiocfg/main.py", line 21, in <module>
    localedir=libcalamares.utils.gettext_path(), 
 
```

Reply:
Sorry that was a typo.